skyfield/data/mpc.py

```diff
@@ -226,6 +226,30 @@ def comet_orbit(row, ts, gm_km3_s2):
     comet._rotation = inertial_frames['ECLIPJ2000'].T
     return comet
 
+def _comet_orbits(rows, ts, gm_km3_s2):
+    e = rows.eccentricity.values
+    p = 1 - e**2
```

Comment:
Surprisingly, it’s generally best to square numbers — in NumPy, and floating point math more generally — by multiplying them by themselves. Try `e*e` instead :slightly_smiling_face:

Reply:
I changed this here, and in a few other places too.